File: 6) C E C++/consumoCombustivel.cpp
#include <iostream>

int main( ) {
	int distancia, consumoMedio, gasto;
	
	std::cout << "Digite a distancia percorrida: ";
	std::cin >> distancia;
	std::cout << "Digite o combustivel gasto: ";
	std::cin >> gasto;
	
	consumoMedio = distancia / gasto;
	
	std::cout << "Consumo medio: " << consumoMedio << std::endl;
}

File: 4) C#/consumoCombustivel.cs
using System;

namespace ConsumoCombustivel
{
	class Program
	{
		static void Main( string[ ] args )
		{
			int distancia, consumo, consumoMedio;
			
			Console.WriteLine( "Digite a distancia percorrida: " );
			distancia = Convert.ToInt32( Console.ReadLine( ) );
			Console.WriteLine( "Digite o combustivel gasto: " );
			consumo = Convert.ToInt32( Console.ReadLine( ) );
			
			consumoMedio = distancia / consumo;
			
			Console.WriteLine( $"Consumo medio: {consumoMedio} " );
		}
	}
}